src/pe_reports/report_generator.py

```diff
@@ -141,70 +152,53 @@ def generate_reports(datestring, output_directory):
+                mi_json,
+                cred_xlsx,
+                da_xlsx,
+                vuln_xlsx,
+                mi_xlsx,
             ) = init(
                 datestring,
                 org_name,
+                org_code,
                 org_uid,
+                premium,
+                output_directory,
+                soc_med_included,
             )
 
             # Convert to HTML to PDF
-            output_filename = f"{output_directory}/{org_code}-Posture_and_Exposure_Report-{datestring}.pdf"
-            convert_html_to_pdf(source_html, output_filename)
-
-            # Create Credential Exposure Excel file
-            cred_xlsx = f"{output_directory}/{org_code}/compromised_credentials.xlsx"
-            credWriter = pd.ExcelWriter(cred_xlsx, engine="xlsxwriter")
-            creds_sum.to_excel(
-                credWriter, sheet_name="Exposed_Credentials", index=False
-            )
-            credWriter.save()
-
-            # Create Domain Masquerading Excel file
-            da_xlsx = f"{output_directory}/{org_code}/domain_alerts.xlsx"
-            domWriter = pd.ExcelWriter(da_xlsx, engine="xlsxwriter")
-            masq_df.to_excel(domWriter, sheet_name="Suspected Domains", index=False)
-            domWriter.save()
-
-            # Create Suspected vulnerability Excel file
-            vuln_xlsx = f"{output_directory}/{org_code}/vuln_alerts.xlsx"
-            vulnWriter = pd.ExcelWriter(vuln_xlsx, engine="xlsxwriter")
-            assets_df.to_excel(vulnWriter, sheet_name="Assets", index=False)
-            insecure_df.to_excel(vulnWriter, sheet_name="Insecure", index=False)
-            vulns_df.to_excel(vulnWriter, sheet_name="Verified Vulns", index=False)
-            vulnWriter.save()
-
-            # Create dark web Excel file
-            mi_xlsx = f"{output_directory}/{org_code}/mention_incidents.xlsx"
-            miWriter = pd.ExcelWriter(mi_xlsx, engine="xlsxwriter")
-            dark_web_mentions.to_excel(
-                miWriter, sheet_name="Dark Web Mentions", index=False
-            )
-            alerts.to_excel(miWriter, sheet_name="Dark Web Alerts", index=False)
-            top_cves.to_excel(miWriter, sheet_name="Top CVEs", index=False)
-            miWriter.save()
+            output_filename = f"{output_directory}/Posture_and_Exposure_Report-{org_code}-{datestring}.pdf"
+            # convert_html_to_pdf(source_html, output_filename)#TODO possibly generate report here
+            report_dict["filename"] = output_filename
+            if premium:
+                report_gen(report_dict, soc_med_included)
+            else:
+                core_report_gen(report_dict)
 
             # Grab the PDF
-            pdf = f"{output_directory}/{org_code}-Posture_and_Exposure_Report-{datestring}.pdf"
+            pdf = f"{output_directory}/Posture_and_Exposure_Report-{org_code}-{datestring}.pdf"
 
-            (filesize, tooLarge) = embed(
+            # Embed excel and Json files
```

Comment:
Use standard capitalization:
```suggestion
            # Embed Excel and JSON files
```